service/crt/service.go

```diff
@@ -29,6 +30,11 @@ const (
 	// resyncPeriod is the period for re-synchronizing the list of objects in k8s
 	// watcher. Set to 1 minute to make the watch more robust.
 	resyncPeriod time.Duration = time.Minute * 1
+
+	TPRName        = "certificate"
```

Comment:
We already have the name in the TPR. I'd like to see this integrated rather than being duplicated here.

https://github.com/giantswarm/certificatetpr/blob/master/name.go